packages/react-components/react-shared-contexts/src/AnnounceContext/AnnounceContext.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+export type AnnounceOptions = {
+  alert?: boolean;
+  batchId?: string;
+  polite?: boolean;
+  priority?: number;
+};
+
+/**
+ * @internal
+ */
+export type AnnounceContextValue<T = AnnounceOptions> = {
+  announce?: (message: string, options?: T) => void;
+};
+
+/**
+ * @internal
+ */
+export const AnnounceContext = React.createContext<AnnounceContextValue | undefined>(undefined);
```

Comment:
```suggestion
const AnnounceContext = React.createContext<AnnounceContextValue | undefined>(undefined);
```

there's no need to export the context itself